addons/survey/views/survey_templates.xml

```diff
@@ -283,12 +283,16 @@
     <template id="survey_fill_form_done" name="Survey: finished">
         <div class="wrap">
             <div class="o_survey_finished mt32 mb32">
-                <h1 class="fs-2">Thank you!</h1>
+                <h1 class="fs-2">
+                    <t t-if="survey.scoring_type != 'no_scoring'">
```

Comment:
Hello @lm4649 ,

Just a question here, I see we will display the score even if we do not have a scoring_success_min, which was not the case before (see condition below). Do we want to change this? I was not sure reading the pad.

You can green if the condition is fine :) Cheers!

Reply:
Hello @Odoonan,

Thanks for checking this PR :pray: 
I guess it's want we want, see [https://github.com/odoo/odoo/pull/165032](url)